deployment/samples/docker-compose/sample-config.yml

```diff
@@ -0,0 +1,47 @@
+nuls2:
+  chain_id: 1
+  enabled: False
+  packing_node: False
+  private_key: YOUR_PRIVATE_KEY_HERE
+  sync_address: NULSd6HgUkssMi6oSjwEn3puNSijLKnyiRV7H
+  api_url: https://apiserver.nuls.io/
+  explorer_url: https://nuls.world/
+  token_contract: NULSd6Hh1FjbnAktH1FFvFnTzfgFnZtgAhYut
+
+ethereum:
+  enabled: True
+  api_url: http://127.0.0.1:8545
+  chain_id: 1
+  packing_node: False
+  private_key: YOUR_PRIVATE_KEY_HERE
+  sync_contract: "0x166fd4299364B21c7567e163d85D78d2fb2f8Ad5"
+  start_height: 11474360
+  token_contract: "0x27702a26126e0B3702af63Ee09aC4d1A084EF628"
+  token_start_height: 10939074
+
+mongodb:
+  uri: "mongodb://mongodb:27017"
+  database: alephtest
+
+storage:
+  store_files: true
+  engine: mongodb
+
+ipfs:
```

Comment:
Why is the field `enabled` absent here ? It may be useful for users to disable IPFS easily.